CODEOWNERS

```diff
@@ -113,8 +113,9 @@
 /src/lib/snark_bits/ @bkase
 /src/lib/snark_work_lib/ @deepthiskumar
 /src/lib/snark_worker/ @cmr @nholland94
+/src/lib/snark_keys/ @imeckler @mrmr1993
 /src/lib/snarkette/ @bkase
-# /src/lib/snarky (submodule!)
+/src/lib/snarky/ @imeckler @mrmr1993
```

Comment:
oh also you don't need to add izzy explicitly to either of these as izzy owns the parent directory (and all subdirectories recursively)